src/events/Custom/timeout.js

```diff
@@ -10,106 +10,128 @@ module.exports = async (d, duration, timeoutData, onReady) => {
-                            const channel = await d.util.getChannel(d, cmd.channel);
-                            if (!channel) return d.aoiError.fnError(d, "custom", {}, `Invalid channel ID in timeout command: ${cmd.channel}`);
+        return;
+    }
 
-                            await d.interpreter(d.client, { channel }, [], cmd, d.client.db, false, undefined, { timeoutData });
-                        } else {
-                            await d.interpreter(d.client, {}, [], cmd, d.client.db, false, undefined, { timeoutData });
-                        }
-                    }
-                    await d.client.db.delete("__aoijs_vars__", `setTimeout_${timeoutData.__id__}`);
-                    clearInterval(interval);
-                },
-                60 * 60 * 1000
-            );
-            await d.client.db.set("__aoijs_vars__", "setTimeout", timeoutData.__id__, { ...timeoutData, __timerID__: interval[Symbol.toPrimitive]() });
-        } else {
-            const timeout = setTimeout(async () => {
-                cmds = cmds.filter((x) => x.name === timeoutData.__timeoutName__);
-                for (const cmd of cmds) {
-                    if (cmd.channel) {
-                        const channel = await d.util.getChannel(d, cmd.channel);
-                        if (!channel) return d.aoiError.fnError(d, "custom", {}, `Invalid channel ID in timeout command: ${cmd.channel}`);
-
-                        await d.interpreter(d.client, { channel }, [], cmd, d.client.db, false, undefined, { timeoutData });
-                    } else {
-                        await d.interpreter(d.client, {}, [], cmd, d.client.db, false, undefined, { timeoutData });
-                    }
+    if (duration > MAX_SAFE_TIMEOUT_DURATION) {
+        const interval = setInterval(async () => {
+            if (timeoutData.__duration__ > Date.now()) return;
+
+            cmds = cmds.filter((x) => x.name === timeoutData.__timeoutName__);
+            for (const cmd of cmds) {
+                if (cmd.channel) {
+                    const channel = await getChannel(d, cmd.channel);
+                    if (!channel) return d.aoiError.fnError(d, "custom", {}, `Invalid channel ID in timeout command: ${cmd.channel}`);
+                    await d.interpreter(d.client, { channel }, [], cmd, d.client.db, false, undefined, { timeoutData });
+                } else {
+                    await d.interpreter(d.client, {}, [], cmd, d.client.db, false, undefined, { timeoutData });
                 }
-                await d.client.db.delete("__aoijs_vars__", `setTimeout_${timeoutData.__id__}`);
-            }, duration);
-            await d.client.db.set("__aoijs_vars__", "setTimeout", timeoutData.__id__, { ...timeoutData, __timerID__: timeout[Symbol.toPrimitive]() });
-        }
+            }
+
+            await d.client.db.delete("__aoijs_vars__", `setTimeout_${timeoutData.__id__}`);
+            clearInterval(interval);
+        }, 3600000);
```

Comment:
if youre replacing that at least use seps for readability


```suggestion
        }, 3_600_000);
```